src/main/kotlin/org/rust/ide/annotator/RsErrorAnnotator.kt

```diff
@@ -109,11 +109,36 @@ class RsErrorAnnotator : AnnotatorBase(), HighlightRangeExtension {
             override fun visitPatTupleStruct(o: RsPatTupleStruct) = checkRsPatTupleStruct(rsHolder, o)
             override fun visitPatTup(o: RsPatTup) = checkRsPatTup(rsHolder, o)
             override fun visitStructLiteralField(o: RsStructLiteralField) = checkReferenceIsPublic(o, o, rsHolder)
+            override fun visitMetaItem(o: RsMetaItem) = checkMetaItem(rsHolder, o)
         }
 
         element.accept(visitor)
     }
 
+    private fun checkMetaItem(holder: RsAnnotationHolder, metaItem: RsMetaItem) {
+        val args = metaItem.metaItemArgs
+        val name = metaItem.name
+        if (args != null && name in listOf("cfg", "cfg_attr")) {
```

Comment:
Let's also check `metaitem.isRootMetaItem()`. And let's add tests for such check:
- a `cfg`/`cfg_attr` that means something non-relevant - `#[foobar(cfg(baz))]`
- a nested `cfg`/`cfg_attr` - `#[cfg_attr(foo, cfg_attr(error_predicate(), baz))]`